airbyte-webapp/src/hooks/services/Analytics/useAnalyticsService.tsx

```diff
@@ -76,6 +76,17 @@ export const useAnalyticsIdentifyUser = (userId?: string, traits?: Record<string
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [userId]);
 };
+export const useAnalyticsSetAnonymousIdFromLocalStorage = (): void => {
+  const analyticsService = useAnalyticsService();
+  const anonymousId = localStorage.getItem("ajs_anonymous_id");
```

Comment:
I'd suggest we move this into a `getSegmentAnonymousId` method into the `crossDomainUtils` file also, so we can keep all the knowledge about how and where it's stored local within this file.